modules/auxiliary/server/ldap_server.rb

```diff
@@ -0,0 +1,89 @@
+##
```

Comment:
None of the other modules really use the `_server` suffix within this folder. They're just named after their protocol.

Reply:
There's a good reason for that - you will see a _proxy server as well.